source/skyconnect/index.html

```diff
@@ -0,0 +1,331 @@
+---
+layout: landingpage
+frontpage: false
+title: "Home Assistant SkyConnect"
+description: "Best way to add Zigbee and Matter to Home Assistant"
+date: 2022-10-05
+tagline: Home Assistant SkyConnect
+tagline_wide: false
+og_image: /images/skyconnect/skyconnect-cover.png
+hero_image: /images/skyconnect/skyconnect_horizontal.png
+frontpage: false
+frontpage_image: /images/skyconnect/skyconnect-cover.png
```

Comment:
```suggestion
```
`frontpage` duplicate, and default.
`frontpage_image` not in use (defined directly in the file above)